pkg/pb/node.proto

```diff
@@ -34,8 +34,12 @@ message Node {
 
 // NodeType is an enum of possible node types
 enum NodeType {
-    ADMIN = 0;
-    STORAGE = 1;
+    INVALID = 0;
+    ADMIN = 1;
```

Comment:
ADMIN is the old name for SATELLITE. we don't need both.

do we ever need UNKNOWN? isn't UNKNOWN just anything above 4? I'd rather not have a specific UNKNOWN type.

Reply:
Yes, Unknown would be for connecting when we don't know what the other side supports. e.g. the bootstrap node.